source/dem/preliminary_force_chain_visualization.cc

```diff
@@ -0,0 +1,95 @@
+
+#include <core/manifolds.h>
+#include <core/solutions_output.h>
+
+#include <dem/data_containers.h>
```

Comment:
Is this file still necessary? You are not using this code, in that case, I would remove it :)